{{cookiecutter.project_slug}}/config/settings/production.py

```diff
@@ -107,6 +107,7 @@
 STATICFILES_STORAGE = "config.settings.production.StaticRootS3Boto3Storage"
 STATIC_URL = f"https://{AWS_STORAGE_BUCKET_NAME}.s3.amazonaws.com/static/"
 {% elif cookiecutter.cloud_provider == 'GCP' -%}
+STATICFILES_STORAGE = "storages.backends.gcloud.GoogleCloudStorage"
```

Comment:
I _beleive_ this will cause Static files to be stored in the same location as Media files (same as `DEFAULT_FILE_STORAGE` + `GS_BUCKET_NAME`).

Ideally, we should create 2 subclasses like we do for AWS below:

https://github.com/pydanny/cookiecutter-django/blob/f3b46ec4251358b7dd5ae24236b7cd9cc0a0b441/%7B%7Bcookiecutter.project_slug%7D%7D/config/settings/production.py#L119-L129

Reply:
oh, cool! I'll update it